### PR TITLE
Add convertToMultiselect helper method

### PR DESCRIPTION
**Pull request by JakeMinor**

### CODE REVIEW

File: packages/ui/src/helpers/index.ts [hunk 1]
```diff
@@ -70,6 +70,13 @@ export function enumToMultiSelectList<T>(items: readonly T[], formatter?: (val:
   return items.map((i) => ({ id: i, label: formatter ? formatter(i) : `${i}` }));
 }
 
+export function convertToMultiSelectOption(id: string, label?: string, shortLabel?: string) {
+  if (!id) {
```
Comment (TomSmith27) [inline packages/ui/src/helpers/index.ts:74]: why do we do this?

Comment (JakeMinor) [reply packages/ui/src/helpers/index.ts:74]: Because if ID is null then it'll just return the options as undefined

Comment (JakeMinor) [reply packages/ui/src/helpers/index.ts:74]: In fairness I think this can be improved by making it generic, just not had the time to look at it again

Comment (TomSmith27) [reply packages/ui/src/helpers/index.ts:74]: But how can I'd be null?

**Review by TomSmith27 — APPROVED** [review]

**Pull request merged**